src/PeriodEnd/LevyPayments/DeployLevyPayments/sql/dml/03 PeriodEnd.Populate.Reference.Commitments.dml.sql

```diff
@@ -47,7 +47,9 @@ INSERT INTO [Reference].[DasCommitments]
         [Priority],
```

Comment:
the "binary" sql files, are they marked as such because they  have sketch line endings? worth checking in notepad++ with show all characters checked. (comment added to this line because I can't comment on the binary lines)